### DISCUSSION TITLE
[11 gennaio 2022 - A] Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#195] [Esami]
**Esercizio 3 (3 punti)**

Usando gli assiomi dell’algebra di Boole, verificare la seguente identità:

![image](https://user-images.githubusercontent.com/69389461/211819927-cd7ac0a8-7140-4eda-b1cf-0b619d67f1c5.png)

**Comment by Elia-Belli** [comment]
- $(\bar a \bar b + ab)+(ac + b) + \overline {a+bc}$
$\bar a \bar b + ab + ac + b + \bar a (\bar b + \bar c)$
$\bar a \bar b + ab + ac + b + \bar a \bar b + \bar a \bar c$
$\bar a \bar b + ac + b + \bar a \bar c$
$(b + \bar b)(b + \bar a) + ac + \bar a \bar c$
$b + \bar a +ac + \bar a \bar c$
$b + (\bar a  + a)(\bar a +c)$
$b + \bar a + c$

**Comment by fraxb04** [comment]
@Elia-Belli 
corretta
